rasa/core/policies/ted_policy.py

```diff
@@ -1524,9 +1578,6 @@ def batch_loss(
 
     # ---PREDICTION---
 
-    def prepare_for_predict(self) -> None:
-        _, self.all_labels_embed = self._create_all_labels_embed()
```

Comment:
why do did you remove this? who will create `self.all_labels_embed`

Reply:
Seems to be not needed anymore, but I am a bit confused why the model is actually training. It is on my list to take a closer look at this.

Reply:
but it is not for training, it is for prediction

Reply:
true 🙈 did not run prediction yet